learning-pathways/sig-and-codex-creation.md

```diff
@@ -0,0 +1,72 @@
+
+  The **Special Interest Group (SIG)** refers to a dedicated scientific community that crosses individual lab boundaries and wants to collaborate, share resources, support each other, and/or collectively advocate on a given theme. We have SIGs based on region, domain of science, and more. You might consider that a SIG covers any group of like-minded Galaxy enthusiasts not currently combined into a Working Group.
+  Here is the list of [currently existing SIGs](https://galaxyproject.org/community/sig/) and the [microgalaxy community of practice webpage](https://galaxyproject.org/community/sig/microbial#microgalaxy-community)
+
+  *References:*
+  Bérénice Batut, Wendi Bacon, Paul Zierep, Matthias Bernt, Nicola Soranzo, et al.. Galaxy CoDex for finding tools, workflows, and training. GCC 2024 - Galaxy Community Conference, Jun 2024, Brno, Czech Republic. ⟨10.7490/f1000research.1119764.1⟩. ⟨hal-04644966⟩
+  Nasr E, Amato P, Bernt M, et al. The Microbiology Galaxy Lab: A community-driven gateway to tools, workflows, and training for reproducible and FAIR analysis of microbial data. Preprint. bioRxiv. 2025;2024.12.23.629682. Published 2025 Mar 26. doi:10.1101/2024.12.23.629682
+
+  **Description of the learning pathway:**
+  This learning pathway is composed of two modules :
+  - Module 1 : The creation of a community CoDex (including the lab interface). Generation of the CoDex includes the generation of three tables : A tool table, a tutorial table and a workflow table.
+  - Module 2: Creation a Special Interest Group (SIG) page (optional).
+
+type: use 
+tags: [community, codex, lab, sig]
+editorial_board:
+- scorreard
+- shiltemann
+- bebatut
+
+cover-image: assets/images/gat.png
+cover-image-alt: Default image
+
+pathway:
+  - section: "Module 1: Create a Galaxy CoDex for your community"
+    description: |
+      The aim is this learning pathway is to create a Galaxy CoDex for your community (including the lab interface).
+      This module is composed of four tutorials :
+        - Tutorial 1 : Creation of resources listing all the tools and their metadata relevant to your community.
+        - Tutorial 2 : Creation of resources listing all the tutorials and their metadata relevant to your community.
+        - Tutorial 3 : Creation of resources listing all the workflows and their metadata relevant to your community.
+        - Tutorial 4 : Creation of the lab interface (in development)
+
+      Tutorials 1, 2 and 3 are stand alones one and they can be done in any order (or skipped altogether).
+      Tutorial 4 requires files generated by tutorials 1, 2 and 3 so should be done last.
+    
+    tutorials:
+      - name: community-tool-table
+        topic: community        
+      - name: community-tutorial-table
+        topic: community
+      - name: community-workflow-table
+        topic: community
+<!---      - name: community-lab
+        topic: community
+-->
```

Comment:
I think your tests are failing because of this commenting, please use # instead

```suggestion
      #- name: community-lab
      #  topic: community

```